crates/parol/Cargo.toml

```diff
@@ -35,6 +35,10 @@ regex-syntax = "0.6.27"
 syn = { version = "1", default-features = false, features = ["full", "parsing"] }
 thiserror = "1.0"
 
+[dev-dependencies]
+# TODO: remove this after examples are updated.
+lazy_static = "1.4"
+
```

Comment:
This is a workaround, and it can be removed after the examples are updated.
I want to fix that we cannot update them automatically except on Windows.
IMO, we have two choices:
1. Convert each of them into a crate like basic_interpreter. Then they are updated automatically with such commands as `cargo test` or `cargo check`. 
2. Use [cargo make](https://github.com/sagiegurari/cargo-make) or any task runner that support cross-platform.

I prefer the 1. If it's okay, I can create a pull request.

Reply:
@jsinger67 How do you think about this?